src/GeoPing.Core/Models/DTO/UserNameAndAvatarDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GeoPing.Core.Models.DTO
+{
+    public class UserNameAndAvatarDTO
```

Comment:
What if we want to load also email and user type? Wouldn't it better to call it ShortUserProfile ?

Reply:
Maybe. We can use this DTO for another tasks, so it may  be renamed, but it seems te be better ShortUserInfoDTO then.